pkg/openstack/loadbalancer.go

```diff
@@ -3292,6 +3305,9 @@ func (lbaas *LbaasV2) updateSecurityGroup(clusterName string, apiService *corev1
 	}
 
 	for _, port := range ports {
+		if port.NodePort == 0 {
```

Comment:
Nit: not sure if you meant to leave a comment about NodePort being 0 here too.

Reply:
Sure, a comment never hurts, done.